src/eve-server/system/SystemBubble.cpp

```diff
@@ -193,62 +193,106 @@ void SystemBubble::ProcessWander(std::vector<SystemEntity*> &wanderers) {
         ResetBubbleRatSpawn();
 }
 
-void SystemBubble::Add(SystemEntity* pSE)
-{
+void SystemBubble::Add(SystemEntity* pSE) {
```

Comment:
everything in this file was just code formatting fixes; no actual changes intended